pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -221,6 +221,10 @@ func getPrimaryInterfaceID(machine compute.VirtualMachine) (string, error) {
 }
 
 func getPrimaryIPConfig(nic network.Interface) (*network.InterfaceIPConfiguration, error) {
+	if nic.IPConfigurations == nil {
```

Comment:
@feiskyer - out of curiosity, do we know why this happens ? No IP config for the network interface of the node ?

Reply:
@yastij Yep, this is because some virtualmachines and network interfaces are removing

Reply:
ok LGTM